src/game/AI/ScriptDevAI/scripts/outland/tempest_keep/botanica/botanica.cpp

```diff
@@ -108,8 +127,8 @@ void instance_botanica::AddInstanceEvent(uint32 id, std::function<bool(Unit cons
     {
         for (const auto& data : instance->GetPlayers())
         {
-            if (data.getSource()->IsGameMaster())
-                continue;
+            // if (data.getSource()->IsGameMaster())
```

Comment:
Ehm - you know you can just do .gm off

Reply:
Yeah, i just removed it so i dont have to kill all those npcs around every time i test it, added it back